frontend/packages/ceph-storage-plugin/src/components/ocs-storage-class-form.tsx

```diff
@@ -0,0 +1,174 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { useK8sWatchResource } from '@console/internal/components/utils/k8s-watch-hook';
+import { cephBlockPoolResource } from '../constants/resources';
+import { Dropdown } from '@patternfly/react-core/dist/js/components/Dropdown/Dropdown';
+import { DropdownItem } from '@patternfly/react-core/dist/js/components/Dropdown/DropdownItem';
+import { DropdownToggle } from '@patternfly/react-core/dist/js/components/Dropdown';
+import { CaretDownIcon } from '@patternfly/react-icons';
+
+const PoolResourceComponent: React.FC<PoolParams> = ({ onParamChange }) => {
+  const [data, loaded, loadError] = useK8sWatchResource<K8sResourceKind[]>(cephBlockPoolResource);
+  const [isOpen, setOpen] = React.useState<boolean>(false);
```

Comment:
```suggestion
  const [isOpen, setIsOpen] = React.useState<boolean>(false);
```

Reply:
Maybe it's my background as a Java programmer, but I have a strong preference for isOpen/setOpen

Reply:
That works! 👍 

When I was reading setOpen it looked to me like its the setter for some other variable not for `isOpen`.